### PR TITLE
Digit DP: Integers with maximum 7 non zero digits ::issue #640

### PR DESCRIPTION
**Pull request by kritigoyal**

### CODE REVIEW

**Comment by jainaman224** [comment]
Please refer the issue you are solving.

File: Add Digit DP using C++ [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include<bits/stdc++.h>
```
Comment (jainaman224) [inline Add Digit DP using C++]: Please fix file name.

**Comment by gupta2140** [comment]
Please include the algorithm name in the description, "Digit DP: Number of integers with maximum 7 nonzero digits".

**Comment by gupta2140** [comment]
Also, this is not digit DP. Please research more on the topic.
If you need any references or resources, don't hesitate to ask!

**Comment by kritigoyal** [comment]
Sorry for disappointing you. It was my mistake that I just read the
description of the problem and coded it without realising that I was
required to implement the algorithm.I will gain more knowledge on this algo
and make changes in the code really soon.

**Comment by kritigoyal** [comment]
Can you please let me know if any more changes are required in this code.

**Comment by jainaman224** [comment]
Hey which issue are you solving. You have added files for two issues. Please separate them and use git branching. If you facing any trouble you can ask on public channel.

**Comment by jainaman224** [comment]
@kritigoyal Let's have a discussion on gitter.

**Pull request closed without merging**